gem/lib/earlgrey/version.rb

```diff
@@ -14,5 +14,5 @@
 #  limitations under the License.
 
 module EarlGrey
-  VERSION = '0.2.1'.freeze unless defined? ::EarlGrey::VERSION
+  VERSION = '0.2.2'.freeze unless defined? ::EarlGrey::VERSION
```

Comment:
can we just make it `1.9.2`? `0.2.2` is not conveying any useful info to the end user